tests/test_claude_wrapper_hooks.py

```
@@ -328,6 +336,263 @@ def decode_nul_argv(encoded: str) -> list[str]:
+                env["FAKE_CHILD_ARGS_JSON"] = json.dumps(child_args)
+            else:
+                env.pop("FAKE_CHILD_ARGS_JSON", None)
+            if child_node_options is not None:
+                env["FAKE_CHILD_NODE_OPTIONS"] = child_node_options
+            else:
+                env.pop("FAKE_CHILD_NODE_OPTIONS", None)
+            env.pop("NODE_OPTIONS", None)
+            env.pop("CMUX_CLAUDE_HOOKS_DISABLED", None)
+
+            proc = subprocess.run(
+                ["claude", "agents"],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)